packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -320,7 +319,9 @@ export const DatePicker = ({
               aria-owns={floatingBoxId}
               data-testid="datepicker-input-button"
               isDisabled={isDisabled}
-              onClick={toggleIsOpen}
```

Comment:
I don't see the problem with this - is there a reason for the change?

Reply:
No idea why that changed wasn't one of mine.